### DISCUSSION TITLE
p/e55a67646bf546b3900ce270a6fbc6ca/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#64] [Announcements]
# p/e55a67646bf546b3900ce270a6fbc6ca/

Greasemonkey 踩坑之路 场景 window 对象不能和外部交换数据 Greasemonkey API 显示 undefined 内存爆炸 Greasemonkey 加载时机太晚 等待一段时间再调用，例如等个几秒 Greasemonkey 脚本可能就加载了 延迟到 Greasemonkey 脚本加载完成再与之交互 暴露出需要交互的函数等到 Greasemonkey 加载完成后进行回调

https://blog.rxliuli.com/p/e55a67646bf546b3900ce270a6fbc6ca/

**Comment by wuxianggujun** [comment]
大佬遇到过，油猴脚本无法再控制台正常输出日志的情况吗？
